package.json

```diff
@@ -36,14 +36,14 @@
     "@types/mocha": "^7.0.0",
     "@types/node": "^10.5.2",
     "@types/sinon": "^7.0.5",
+    "c8": "^7.0.0",
     "codecov": "^3.0.4",
-    "gts": "^1.0.0",
+    "gts": "2.0.0-alpha.5",
     "mocha": "^7.0.0",
-    "c8": "^7.0.0",
     "sinon": "^9.0.0",
-    "typescript": "3.6.4"
+    "typescript": "^3.8.3"
   },
   "engines": {
-    "node": ">=10"
+    "node": ">=10.4.0"
```

Comment:
Is this the story across the board?  `10.4.0`?

Reply:
BigInt is suitable until 10.4.0